.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
     "trailingComma": "es5",
-    "tabWidth": 4,
+    "tabWidth": 2,
     "semi": false,
     "singleQuote": true
   }
```

Comment:
No newline at end of file might cause a linting error

Reply:
You want me to add a prettier rule to always add a newline at EOF u mean?

Reply:
Oh, if there is no rule for that then it should be OK.